src/dft/CMakeLists.txt

```diff
@@ -32,13 +32,13 @@ target_include_directories(onemkl_dft
           $<TARGET_FILE_DIR:onemkl>
 )
 
-target_compile_options(onemkl_dft PRIVATE ${ONEMKL_BUILD_COPT})
+target_compile_options(onemkl_dft PRIVATE ${ONEMKL_BUILD_COPT} -DBUILD_RUN)
```

Comment:
this is mostly to selectively build the descriptor.cpp so that in the
libonemkl.so only the detail::create_commit symbol is defined
libonemkl_dft_mklxpu.so only the xpu::create_commit is defined